docker/Dockerfile

```diff
@@ -2,8 +2,8 @@ ARG dev_cycle=develop
 FROM lsstts/develop-env:${dev_cycle}
 
 WORKDIR /usr/src/love
-COPY simulator/requirements.txt .
-RUN source /opt/lsst/software/stack/loadLSST.bash && pip install -r requirements.txt
+RUN source /home/saluser/.setup_dev.sh && \
+  pip install gevent greenlet six websocket-client websockets eventlet
```

Comment:
So `requirements.txt` was not unused after all :-) Any plans to switch this Dockerfile to conda? I checked and all packages are available in coda as well. Also, installing packages without a pin on their versions generally is not recommended. I'll approve the PR but please try to think about all of this for a future ticket.